src/components/UI/MovieCategoryDropdown.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="dropdown-categories">
+    <label class="dropdown-categories__label" for="">Categories</label>
+    <select
+      class="dropdown-categories__select"
+      @change="$store.commit('setSelectedGenre', $event.target.value)"
+    >
+      <option
+        class="dropdown-categories__options"
+        v-for="genre in ['All Categories', ...$store.getters.genreList]"
+        :key="genre"
+      >
+        {{ genre }}
+      </option>
+    </select>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "MovieCategoryDropdown",
+  props: {
+    listOptions: {
+      type: Array,
+      default: null,
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.dropdown-categories {
+  width: 100%;
+
+  &__label {
+    font-family: Roboto, Mono;
```

Comment:
I see that we use this font in several different places, maybe there could be a better way to unify our font families?
Maybe introduce a variable to keep font-family definitions in one place? 🤔

Reply:
This makes sense if its going to be repeating. I have included defition for this font style in mixins (I hope that is what you had in mind) and will keep on adding to it as I go along. Thank you 😺